algebra/src/main/java/org/almostrealism/collect/CollectionFeatures.java

```diff
@@ -143,6 +243,31 @@ default TraversalPolicy shape(TraversableExpression t) {
 		}
 	}
 
+	/**
+	 * Gets the total number of elements that a Supplier can produce.
```

Comment:
This is not the total number of elements that can be produced, but rather the number that will be operated on by one thread of a kernel. It is the product of the size and the count that determines the total.